packages/nuxt/src/core/nuxt.ts

```diff
@@ -251,7 +251,8 @@ async function initNuxt (nuxt: Nuxt) {
       name: 'NuxtClientFallback',
       priority: 10, // built-in that we do not expect the user to override
       filePath: resolve(nuxt.options.appDir, 'components/client-fallback.client'),
-      mode: 'client'
+      mode: 'client',
+      noClientOnlyTransform: true
```

Comment:
Is there a better naming for this ?

Reply:
prewrapped? raw?

Reply:
raw might be confusing since vue has markRaw which you might use on a component https://vuejs.org/api/reactivity-advanced.html#markraw

Reply:
oh i'd prefer raw over prewrapped 😅